setup.cfg

```diff
@@ -228,10 +228,12 @@ pretty = False
 # Warns about per-module sections in the config file that do not match any files processed when invoking mypy.
 warn_unused_configs = True
 # Make arguments prepended via Concatenate be truly positional-only.
-strict_concatenate = True
+extra_checks = True
```

Comment:
May also update the requirement for mypy in `requirements-dev.txt`.
I checked that `extra_checks` was introduced from 1.5.0